src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: -2px 4px 4px #ececef;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-right: 20px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+  font-size: 12px;
+  color: #000;
```

Comment:
The nav link in default state has different color change it to `#060b35` and you will pass all the tests.